sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightPlugin.kt

```diff
@@ -81,20 +82,24 @@ abstract class SqlDelightPlugin : Plugin<Project> {
 
     val isMultiplatform = project.plugins.hasPlugin("org.jetbrains.kotlin.multiplatform")
 
+    val needsSyncRuntime = extension.databases.any { !it.generateAsync }
+    val needsAsyncRuntime = extension.databases.any { it.generateAsync }
+    val runtimeDependencies = mutableListOf<Dependency>().apply {
```

Comment:
```suggestion
    val runtimeDependencies = buildList {
```

I think this is an API that exists

Reply:
`java-gradle-plugin` depends on Kotlin 1.5.31 where that function is still experimental, otherwise I would have used it.